subprojects/docs/src/docs/userguide/authoring-builds/tutorial/partr2_build_lifecycle.adoc

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[partr2_build_lifecycle]]
+= Part 2: The Build Lifecycle
+
+Learn about the Gradle build lifecycle and what each phase represents.
+
+****
+**In this section you will:**
+
+- Understand the Build Lifecycle
+- Register and configure two tasks
+- Run the task to view the phases in action
+****
+
+[[part2_begin]]
+== Step 0. Before you Begin
+
+1. You initialized your Java app in <<partr1_gradle_init.adoc#part1_begin,part 1>>.
+
+== Step 1. Understanding the Build Lifecycle
+
+A Gradle build has three distinct phases:
+
+Phase 1 - Initialization  :: During the initialization phase, Gradle determines which projects are going to take part in the build, and creates a `Project` instance for each of these projects.
+
+Phase 2 - Configuration  :: During this phase, the `Project` objects are configured using the build scripts of all projects which are part of the build are executed.
+
+Phase 3 - Execution  :: Gradle determines the set of tasks, created and configured during the configuration phase, to be executed. Gradle then executes each of the selected tasks.
```

Comment:
❗ Gradle determines the set of tasks to be executed during the configuration phase.